packages/tenant-process/test/tenant.integration.test.ts

```diff
@@ -153,17 +157,26 @@ describe("Integration tests", () => {
           attributeId,
           updateVerifiedTenantAttributeSeed,
         });
-        const writtenEvent = await readLastEventByStreamId(
-          tenant.id,
-          postgresDB
-        );
+        const writtenEvent: StoredEvent | undefined =
+          await readLastEventByStreamId(
+            tenant.id,
+            eventStoreSchema.tenant,
+            postgresDB
+          );
+        if (!writtenEvent) {
+          fail("Creation fails: tenant not found in event-store");
+        }
+        expect(writtenEvent).toBeDefined();
```

Comment:
this is a duplicated check, isn't the `expect` enough?

Reply:
Without the if statement there is a compilation error: `writtenEvent` is probably undefined, but they are actually the same control, so I would suggest removing the `expect`.
Both are present in the agreement tests.
If you wanted to remove the if statement you would have to force the `writtenEvent` with the `?`
How would it be better?